### ISSUE TITLE
installer failures

### ISSUE THREAD

**Issue by Televicious** [henrikstengaard/hstwb-installer#25]
So I've made a bunch of images with the installer. It used to work fine. lately it's not been installing the actual launcher or when it does it says AGS2 isn't installed even though it was in the installer process. Tried all sorts of different ways. Also had to use portable because .msi puts it in program files which is write protected and you can't download the latest packages since the default folder to work in is the one in program files x86. Also having issues making user packages, Was trying to automate the whdload collection after having organized it and matched all the naming. It says the userpackage isn't there though it does find it when I select it to install. Anyhow, been really buggy lately. Maybe I'll go back to 1.2 and see if that is doing the same thing. It didn't used to act like this.

**Comment by henrikstengaard** [comment]
I agree that Program Files is a bad place install HstWB Installer with the download packages functionality. I will change the msi installer to install into users AppData instead of Program Files due to solve permission issues.

To help solving your installation issues, I need more details about what Amiga OS you used, packages selected etc. with short description of steps you did to reproduce and investigate the issue.

The only requirement for user packages, is that they contain a "_installdir" file with directory path to where it should be installed. If this file doesn't exist, then HstWB Installer won't detect the user package.

**Comment by henrikstengaard** [comment]
HstWB Installer v1.3.0 has just been released as portable version only due to the issues present with MSI installing to program files.

Did you examine if "_installdir" files are present in "eab-whdload-demos", "eab-whdload-games", "eab-whdload-games_beta"?

Otherwise please give the latest version a try.
